### PR TITLE
Smcculloch.home screen edits

### PR DESCRIPTION
**Pull request by sjmcculloch**
- Applied new styling to home screen (replaced by facebook notification style feed later):

image

image

- Updated ListItem component to support new look

- Wired home screen elements to pull data from graphql

- Deep Linked items on the home screen (events & requests)

- Created new segment component that wraps a 3rd party control:

image

- Moved home screens on to their own stack

- Removed unused fixtures

### CODE REVIEW

File: client/src/screens/home/Index.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+                  </Center>
+                ) : null)}
+              keyExtractor={schedule => schedule.id}
+              renderItem={schedule => (
+                <ListItem
+                  title={schedule.item.name}
+                  subtitle={
+                    schedule.item.startTime === 0
+                      ? 'Ongoing'
+                      : `${moment(schedule.item.startTime).format('YYYY-MM-DD')} to ${moment(
```
Comment (OSPFNeighbour) [inline client/src/screens/home/Index.js]: this doesn't seem to be working, everything is coming up 1970

Comment (sjmcculloch) [reply client/src/screens/home/Index.js]: Hmm I thought it was just the test data you loaded - I'll fix now.

**Comment by OSPFNeighbour** [comment]
Android images attached. Tested working on Android.

screenshot 2018-02-09 10 19 51
screenshot 2018-02-09 10 19 45

File: client/src/screens/home/Index.js (hunk 1 above)
Comment (sjmcculloch) [reply client/src/screens/home/Index.js]: Should be fixed

Comment (OSPFNeighbour) [reply client/src/screens/home/Index.js]: {
          "name": "Wagga Wagga OOA",
          "startTime": 1517837709,
          "endTime": 1518010509
        },

should be, errrm, 

    startTime: MONDAY + (60 * 60 * 24 * 1),
    endTime: MONDAY + (60 * 60 * 24 * 3),

Comment (OSPFNeighbour) [reply client/src/screens/home/Index.js]: That fixed it.

**Pull request merged**